ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+Hello, thanks for opening an issue in our project!  
```

Comment:
This can be enclosed in a single comment instead of text so it doesn't show up in issues using `<!-- ... -->`. Same for all other text.

Reply:
Comments is a good idea, I didn't think of that!